src/components/common/FormButton.js

```diff
@@ -321,13 +322,19 @@ const FormButton = ({
     disabled = false,
     onClick,
     sending = false,
-    size = ''
+    size = '',
+    to,
+    history,
+    className
 }) => (
     <CustomButton
         type={type}
-        className={`${color} ${size} ${sending ? `dots` : ``}`}
+        className={`${color} ${size} ${sending ? `dots` : ``} ${className}`}
```

Comment:
![Screen Shot 2020-04-08 at 1 23 42 PM](https://user-images.githubusercontent.com/24921205/78830055-35a3ae80-799c-11ea-8dc0-5213894df94e.png)

I think we should avoid `undefined` if no className?

Reply:
also maybe we should also remove  dependency on `import { Button } from 'semantic-ui-react'` and avoid dealing with `className` here altogether? 

otherwise looks like good use for `utils/classNames`

Reply:
Agree. Let's just make sure we test a lot if we remove semantic-ui since there are a lot of variations of the `FormButton` that may be affected.

Good point on `classNames`

Reply:
>avoid dealing with className here altogether?

@vgrichina what do you mean?

Reply:
I mean that classes need to be set only because we rely on semantic-ui-react.  If not using semantic-ui-react (creating our custom component from scratch) we can for example use props directly in styled-components CSS (or otherwise apply styles programmatically).